doc/sample-nginx.conf

```
@@ -0,0 +1,62 @@
+worker_processes  1;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    # Blockdaemon with bearer token authentication
+    server {
+        listen 7900;
+
+        location / {
+            proxy_pass https://yourserver-solana-01.bdnodes.net;
+            proxy_http_version 1.1;
+            proxy_set_header Host $proxy_host;
+            proxy_set_header Authorization "Bearer ReplaceMeWithYourAuthToken";
+            proxy_redirect https://$proxy_host http://$server_name:$server_port;
+            proxy_redirect https:// http://;
+        }
+    }
+
+    # Figment RPC with auth
+    server {
+        listen 7901;
+
+        location / {
+            proxy_pass https://solana--mainnet--rpc.datahub.figment.io/apikey/ReplaceMeWithYourAuthToken;
+            proxy_http_version 1.1;
+            proxy_set_header Host $proxy_host;
+            proxy_redirect https://$proxy_host http://$server_name:$server_port;
+            proxy_redirect https:// http://;
+        }
+    }
+
+    # Figment websocket with auth
+    server {
+        listen 7902;
+
+        location / {
+            proxy_pass https://solana--mainnet--ws.datahub.figment.io/apikey/ReplaceMeWithYourAuthToken;
```

```suggestion
            proxy_pass https://solana--mainnet.datahub.figment.io/apikey/ReplaceMeWithYourAuthToken;
```

See my other comment above.